modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/reducers/fieldSetReducer.js

```diff
@@ -156,7 +156,7 @@ export default (state, action, config) => {
 			const fieldSetId = `${id}`;
```

Comment:
Hi @danvb112, I believe removing `id` cast is better than adding cast in `ddmStructureId`

Reply:
Hi @EliveltonR18 , I tried this before, but in the case of the saved structure, this cast in `id` is important because the `ddmStructureId` always returns as String except for the case of unsaved structures that the return is an Integer. If I remove this cast in `id`, I would have to do a cast in `ddmStructureId` to ensure it will be an integer

Reply:
I understand, @danvb112!